app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx

```diff
@@ -1,41 +1,48 @@
 import React, { useState } from "react";
 
 import { Formik, Form } from "formik";
-import { Button, Pane } from "neetoui";
+import { Button, Pane, Toastr } from "neetoui";
 import { Input, Select } from "neetoui/formik";
 
 import { CONTACTS_FORM_VALIDATION_SCHEMA } from "../constants";
 
 const ContactForm = ({ onClose, contact }) => {
   const [submitted, setSubmitted] = useState(false);
 
+  const handleSubmit = () => {
+    try {
+      Toastr.success("Contact added successfully.");
+      onClose();
+    } catch (err) {
+      Toastr.success("Failed to add contact.");
+      logger.error(err);
+    }
+  };
+
   return (
     <Formik
       initialValues={contact}
       validateOnBlur={submitted}
       validateOnChange={submitted}
       validationSchema={CONTACTS_FORM_VALIDATION_SCHEMA}
-      onSubmit={() => {}}
+      onSubmit={() => handleSubmit()}
```

Comment:
Use the shorthand notation whenever possible.

```suggestion
onSubmit={handleSubmit}
```

Reply:
I have resolved this.